docs/docsite/rst/dev_guide/developing_python_3.rst

```diff
@@ -1,34 +1,30 @@
 .. _developing_python_3:
 
-====================
-Ansible and Python 3
-====================
+********************
+Ansible and Python-3
+********************
 
-Ansible is pursuing a strategy of having one code base that runs on both
+.. contents:: Topics
+   :local:
+
+Ansible maintains a single code base that runs on both
 Python-2 and Python-3 because we want Ansible to be able to manage a wide
 variety of machines.  Contributors to Ansible should be aware of the tips in
 this document so that they can write code that will run on the same versions
 of Python as the rest of Ansible.
 
-Ansible can be divided into three overlapping pieces for the purposes of
-porting:
-
-1. Controller-side code.  This is the code which runs on the machine where you
-   invoke :command:`/usr/bin/ansible`
-2. Modules.  This is the code which Ansible transmits over the wire and
-   invokes on the managed machine.
-3. module_utils code.  This is code whose primary purpose is to be used by the
-   modules to perform tasks.  However, some controller-side code might use
-   generic functions from here.
+To ensure that your code runs on Python-3 as well as on Python-2, learn the tips and tricks and idioms
```

Comment:
I don't believe we need the dashes between Python and the version number. Just "Python 2" and "Python 3".